catalog/dags/database/staging_database_restore/staging_database_restore.py

```diff
@@ -207,3 +209,43 @@ def make_rename_task_group(
         rename >> await_rename
 
     return rename_group
+
+
+@task
+@setup_github
+def get_latest_api_package_version(github: GitHubAPI = None) -> str:
+    """Get the latest version of the API package from GitHub."""
+    versions = github.get_package_versions("openverse-api")
+    tags = set(versions[0]["metadata"]["container"]["tags"])
+    # There might actually be more than one tag here, but since they all point
+    # to the same build we don't care which is used.
+    latest_version = sorted(tags - {"latest"})[0]
+    log.info(f"Found latest version: {latest_version}")
+    if not ("latest" in tags or latest_version.startswith("rel-")):
+        raise ValueError(
+            f"Latest version is not a release and not marked with 'latest': "
```

Comment:
```suggestion
            f"Latest version is not a release or not marked with 'latest': "
```
Just making sure: this error will not be raised if only one of these is false (i.e., either it must have the `latest` tag _or_ it must begin with `rel-`). Based on the tests it looks like that's the desired behavior, but the error message initially made me think the logic might be wrong.

Reply:
Ah, yes, thanks for the correction!